DDR4 OC Guide.md

```diff
@@ -37,30 +37,28 @@
-* Make sure your CPU is fully stable before overclocking RAM, as an unstable CPU can lead to memory errors. When pushing high frequency with tight timings, it's possible that your CPU can become unstable.
-* Make sure your UEFI is up to date.
-
 ## Memory Testing Software
-You should always test with a variety of stress tests to ensure your overclock is stable.
+You should always test with various stress tests to ensure your overclock is stable.
 ### Avoid
-* I wouldn't recommend AIDA64 memory test and [Memtest64](https://forums.anandtech.com/threads/techpowerups-memtest-64-is-it-better-than-hci-memtest-for-determining-stability.2532209/) as they are both not very good at finding memory errors.
+* I wouldn't recommend the AIDA64 memory test and [Memtest64](https://forums.anandtech.com/threads/techpowerups-memtest-64-is-it-better-than-hci-memtest-for-determining-stability.2532209/) as they are both not very good at finding memory errors.
 ### Recommended
 * [TM5](http://testmem.tz.ru/tm5.rar) with any of the configs listed:
-  * [Extreme config by anta777](https://drive.google.com/file/d/1uegPn9ZuUoWxOssCP4PjMjGW9eC_1VJA) (recommended). Make sure to load the config. It should say 'Customize: Extreme1 @anta777' if loaded.  
+  * [Extreme config by anta777](https://github.com/integralfx/MemTestHelper/blob/oc-guide/extreme%40anta777.cfg) (recommended). Make sure to load the config. It should say 'Customize: Extreme1 @anta777' if loaded.  
   Credits: [u/nucl3arlion](https://www.reddit.com/r/overclocking/comments/dlghvs/micron_reve_high_training_voltage_requirement/f4zcs04/).
-  * [Here](https://www.overclock.net/threads/memory-testing-with-testmem5-tm5-with-custom-configs.1751608/) is a link to TM5 pre-packaged with many configs.
-  * [LMHz Universal 2 config](https://www.hardwareluxx.de/community/threads/ryzen-ram-oc-m%C3%B6gliche-limitierungen.1216557/page-159#post-27506598)
-  * If you experience issues with all threads crashing upon launch with the extreme config it might help to edit the row "Testing Window Size (Mb)=1408". Replace the window size with your total RAM (minus some margin for Windows) divided by your processors available threads (e.g. 12800/16 = 800 MB per thread).
-* [OCCT](https://www.ocbase.com/) with the dedicated memory test using either SSE or AVX instructions.
-  * Note that AVX and SSE can vary in error detection speed. On Intel-based systems, SSE appears better for testing IMC voltages while AVX appears better for DRAM voltage.
-  * The Large AVX2 CPU test is a great stability test for your CPU and RAM at the same time. The more you tune your ram the harder it'll be to stable in this test.
+  * [Absolut](https://github.com/integralfx/MemTestHelper/blob/oc-guide/absolutnew.cfg)
+  * [PCBdestroyer](https://github.com/integralfx/MemTestHelper/blob/oc-guide/PCBdestroyer.cfg)
+  * [LMHz Universal 2 config](https://github.com/integralfx/MemTestHelper/blob/oc-guide/Universal-2%40LMhz.cfg)
```

Comment:
These configs don't exist in the repo

You need to add it to your repo and use a relative link
e.g. `TM5/extreme%40anta777.cfg`

Reply:
I was thinking of the files being added to your repo directly but that may be too messy with all the files. I'll go do that.

Reply:
You can't add the files directly to my repo anyway.

You need to add them to yours so that the changes get merged into my repo.

I'd recommend you read into git branching: https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow

Reply:
I have added relative links to all the files.